### PR TITLE
Stop swallowing exceptions in the $everything operation

### PR DESCRIPTION
**Pull request by lmsurpre**
Signed-off-by: Lee Surprenant <lmsurpre@us.ibm.com>

### CODE REVIEW

File: operation/fhir-operation-everything/src/main/java/com/ibm/fhir/operation/everything/EverythingOperation.java [hunk 1]
```diff
@@ -250,7 +258,8 @@ protected Parameters doInvoke(FHIROperationContext operationContext, Class<? ext
         try {
             outputParameters = FHIROperationUtil.getOutputParameters(bundleBuilder.build());
         } catch (Exception e) {
-            FHIROperationException exceptionWithIssue = buildExceptionWithIssue("An unexpected error occurred while creating the operation output parameters for the resulting Bundle.", IssueType.EXCEPTION);
+            FHIROperationException exceptionWithIssue = buildExceptionWithIssue("An unexpected error occurred while "
+                    + "creating the operation output parameters for the resulting Bundle.", IssueType.EXCEPTION, e);
```
Comment (prb112) [inline operation/fhir-operation-everything/src/main/java/com/ibm/fhir/operation/everything/EverythingOperation.java:262]: I believe the PR review we asked that Luis not send back the exceptions.  This is a bit of a design change.  
I think this deserves a bit of javadoc explaining the choice so we don't go revisit this in the future and flip the other way.

Comment (lmsurpre) [reply operation/fhir-operation-everything/src/main/java/com/ibm/fhir/operation/everything/EverythingOperation.java:262]: Are you thinking javadoc at each location where we catch + wrap?   Or just mention it once at the class level?

Comment (prb112) [reply operation/fhir-operation-everything/src/main/java/com/ibm/fhir/operation/everything/EverythingOperation.java:262]: I was just thinking at the class level.

**Review by prb112 — APPROVED** [review]
LGTM, (adding the Javadoc is just a nice to have)

File: operation/fhir-operation-everything/src/main/java/com/ibm/fhir/operation/everything/EverythingOperation.java (hunk 1 above)
Comment (lmsurpre) [reply operation/fhir-operation-everything/src/main/java/com/ibm/fhir/operation/everything/EverythingOperation.java:262]: OK, how about I add something to our style guide at https://github.com/IBM/FHIR/wiki/Style-Guide about it instead?
We already cover exception handling there, so I could add a blurb about "wrap, rethrow, log and throw new, or handle; don't swallow unless you handle it"?

**Pull request merged**